sapi/cli/php_cli_server.c

```diff
@@ -1629,6 +1629,9 @@ static int php_cli_server_client_read_request_on_path(php_http_parser *parser, c
 	{
 		char *vpath;
 		size_t vpath_len;
+		if (client->request.vpath != NULL) {
+			return 1;
```

Comment:
It may be good to test this case -- I wouldn't trust us to handle failure conditions correctly :)

Reply:
Good idea! The failure isn't handled really bad, but we may consider to provide an actual 4xx response in the future (now the server just logs an error to stderr, and closes the connection).